homeassistant/components/http/auth.py

```diff
@@ -192,7 +192,7 @@ def setup_auth(hass, app):
               await async_validate_trusted_networks(request)):
             _LOGGER.warning(
                 'Using trusted networks to bypass authentication is going to'
-                ' deprecate after 0.95 release. You need to use a bearer token'
+                ' removed after 0.95 release. You need to use a bearer token'
```

Comment:
Should say `is going to be removed after 0.95 release`

Should we also include a link to some docs with more information?